inc/api.class.php

```diff
@@ -257,30 +257,11 @@ protected function retrieveSession() {
          $current = session_id();
          $session = trim($this->parameters['session_token']);
 
-         if (file_exists(GLPI_ROOT . '/inc/downstream.php')) {
```

Comment:
You can't drop all of this. See http://glpi-install.readthedocs.io/en/latest/install/index.html#files-and-directories-locations

Reply:
I will revert